server/api/classyear/classyear.controller.js

```diff
@@ -12,7 +12,7 @@ exports.index = function(req, res) {
   	"current": true
   }, function (err, classYear) {
   	if(err) { return handleError(res, err); }
-   	return res.json(200, classYear);
+   	return res.status(200).json(classYear);
```

Comment:
Depreciated function.  Didn't want to make a full PR just for a couple line changes